### PR TITLE
enable debian

### PR DESCRIPTION
**Pull request by bodsch**
I've tested icingaweb2 with my debian-node an add some changes for this

### CODE REVIEW

**Comment by arioch** [comment]
Almost there.

'enable debian' is a little confusing. Could you amend it to  'Add Debian support' or similar?

Last thing that's missing is spec tests. Have a look at [icingaweb2_spec.rb](https://github.com/Icinga/puppet-icingaweb2/blob/master/spec/classes/icingaweb2_spec.rb) for inspiration.

File: manifests/preinstall/debian.pp [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# == Define: icingaweb2::preinstall::redhat
```
Comment (arioch) [inline manifests/preinstall/debian.pp]: There's a typo on the first line. 

```# == Define: icingaweb2::preinstall::redhat``` should be ```# == Define: icingaweb2::preinstall::debian```

**Comment by bodsch** [comment]
Sorry about my long abstinence.
I'm the technical maintainer for a german news-site and we had an heavy relaunch for a couple of weeks.

**Comment by bodsch** [comment]
good morning!

i have corrected the git commit and change the debian.pp with arloch recommendation.
I hope t'ts so better. ;)

Greetings from Hamburg,
 Bodo

**Comment by arioch** [comment]
```
manifests/preinstall/debian.pp - WARNING: top-scope variable being used without an explicit namespace on line 14
manifests/preinstall/debian.pp - WARNING: top-scope variable being used without an explicit namespace on line 20
```

Could fix those errors and rebase your commits into a single commit?

**Comment by lazyfrosch** [comment]
Thanks for your work @bodsch 

I'm adopting the repository behavior of `icinga/icinga2`. See #72

**Pull request closed without merging**